### PR TITLE
Add UI for submit payment details

### PR DESCRIPTION
**Pull request by yg-apaza**

### CODE REVIEW

**Comment by yg-apaza** [comment]
@TimMoore @ignasi35 please review :) and also #151

**Comment by ignasi35** [comment]
Hi @yg-apaza sorry for the delay. Reviewing...

File: web-gateway/src/main/resources/routes [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+GET         /transaction/:id                        controllers.TransactionController.getTransaction(id)
+GET         /transaction/:id/delivery               controllers.TransactionController.submitDeliveryDetailsForm(id)
+POST        /transaction/:id/delivery               controllers.TransactionController.submitDeliveryDetails(id, transactionStatus, isBuyer: Boolean)
+GET         /transaction/:id/deliveryprice          controllers.TransactionController.setDeliveryPriceForm(id)
+POST        /transaction/:id/deliveryprice          controllers.TransactionController.setDeliveryPrice(id, transactionStatus, isSeller: Boolean)
+POST        /transaction/:id/approvedelivery        controllers.TransactionController.approveDelivery(id)
+GET         /transaction/:id/payment                controllers.TransactionController.submitPaymentDetailsForm(id)
+POST        /transaction/:id/payment                controllers.TransactionController.submitPaymentDetails(id, transactionStatus, isBuyer: Boolean)
```
Comment (ignasi35) [inline web-gateway/src/main/resources/routes]: I'm not sure there's any change here other than formatting.

Comment (ignasi35) [inline web-gateway/src/main/resources/routes]: Oh, wait! `payment` is added.

**Pull request merged**